cmd/tsgo/main.go

```diff
@@ -120,7 +123,23 @@ func parseArgs() *cliOptions {
 	return opts
 }
 
+func enableVirtualTerminalProcessing() {
+	hStdout, err := syscall.GetStdHandle(syscall.STD_OUTPUT_HANDLE)
```

Comment:
As of Go 1.17+, the syscall package is considered deprecated for many use cases in favor of golang.org/x/sys/windows. Your code already imports golang.org/x/sys/windows, so you can use this instead:
```go
hStdout, err := windows.GetStdHandle(windows.STD_OUTPUT_HANDLE)
```